README.md

```diff
@@ -32,18 +32,30 @@ Follow instructions in http://kafka.apache.org/documentation.html#quickstart
     ./gradlew scaladocJar # builds a scaladoc jar for each module
     ./gradlew docsJar # builds both (if applicable) javadoc and scaladoc jars for each module
 
-### Running unit tests ###
-    ./gradlew test
+### Run unit tests ###
+    ./gradlew unitTest
 
+### Run integration tests ###
+    ./gradlew integrationTest
+    
+### Run unit & integration tests ###
+    ./gradlew test
+    
 ### Forcing re-running unit tests w/o code change ###
+    ./gradlew cleanTest unitTest
+
+### Forcing re-running integration tests w/o code change ###
+    ./gradlew cleanTest integrationTest
+
+### Forcing re-running unit & integration tests w/o code change ###
     ./gradlew cleanTest test
 
 ### Running a particular unit test ###
-    ./gradlew -Dtest.single=RequestResponseSerializationTest core:test
+    ./gradlew -Dtest.single=RequestResponseSerializationTest core:unitTest
 
 ### Running a particular test method within a unit test ###
-    ./gradlew core:test --tests kafka.api.ProducerFailureHandlingTest.testCannotSendToInternalTopic
-    ./gradlew clients:test --tests org.apache.kafka.clients.MetadataTest.testMetadataUpdateWaitTime
+    ./gradlew core:unitTest --tests kafka.api.ProducerFailureHandlingTest.testCannotSendToInternalTopic
+    ./gradlew clients:unitTest --tests org.apache.kafka.clients.MetadataTest.testMetadataUpdateWaitTime
```

Comment:
Seems like `core:test` is better since if you are running a particular test, you don't care if it's unit or integration? I'd change `within a unit test` to `within a unit/integration test` in the header instead.

I'd say the same for the `Running a particular unit test` section. We can simply say `Running a particular unit/integration test`.